src/commonMain/kotlin/exchange.dydx.abacus/processor/launchIncentive/LaunchIncentiveProcessor.kt

```diff
@@ -0,0 +1,49 @@
+package exchange.dydx.abacus.processor.launchIncentive
+
+import exchange.dydx.abacus.processor.base.BaseProcessor
+import exchange.dydx.abacus.processor.wallet.account.LaunchIncentivePointsProcessor
+import exchange.dydx.abacus.protocols.ParserProtocol
+import exchange.dydx.abacus.utils.mutable
+
+internal class LaunchIncentiveProcessor(parser: ParserProtocol) : BaseProcessor(parser) {
+    private val seasonsProcessor = LaunchIncentiveSeasonsProcessor(parser = parser)
+    private val pointsProcessor = LaunchIncentivePointsProcessor(parser = parser)
+
+    internal fun receivedSeasons(
+        existing: Map<String, Any>?,
+        payload: Map<String, Any>
+    ): Map<String, Any>? {
+        /*
+        Sample data:
+        {
+           "data":{
+              "tradingSeasons":[
+                 {
+                    "startTimestamp":1701177710,
+                    "label":"1",
+                    "__typename":"TradingSeason"
+                 },
+                 {
+                    "startTimestamp":1704384000,
+                    "label":"2",
+                    "__typename":"TradingSeason"
+                 }
+              ]
+           }
+        }
+         */
```

Comment:
since this is in the test data file, should we remove? Unless you think it is helpful to see in-line with the processor code